OracleDatabase/RAC/OracleDNSServer/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2021 Oracle and/or its affiliates.  All rights reserved.
```

Comment:
We no longer add "All rights reserved." so that should be removed from all the copyright statements. Also, this file is not required as we have the same license declaration in the root of the repository. If you want to retain the license, please rename the file to `LICENSE.txt` and ensure the formatting is _identical_ (down to the location of the line breaks) from the canonical version at <https://opensource.oracle.com/licenses/upl/>